test/Makefile

```diff
@@ -65,3 +65,14 @@ clean:
 	@$(ECHO_CLEAN)
 	-$(QUIET) rm -rf $(TEST_ARTIFACTS)
 	-$(QUIET) rm -f .vagrant/*.box
+
+compile_bpf_tests:
+	$(QUIET) make -C ../bpf/tests/bpf_tests clean
```

Comment:
Does this need to clean every time? If the targets are set up with dependencies correctly, then these should compile iff something actually changed.